internal/evmreader/input.go

```diff
@@ -39,18 +85,27 @@ func (r *Service) checkForNewInputs(
 			"inputbox_address", inputBoxAddress,
 			"most recent block", mostRecentBlockNumber,
 		)
-		appsByLastInputCheckBlock := indexApps(byLastInputCheckBlock, inputBoxApps)
```

Comment:
With this change, `byLastInputCheckBlock` would no longer be used.
Do you think it will be used in the future, and have decided to keep it?